pkg/sdk/go/groups.go

```diff
@@ -11,17 +11,25 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/mainflux/mainflux/auth"
 	"github.com/mainflux/mainflux/pkg/errors"
 )
 
-const groupsEndpoint = "groups"
+const (
+	groupsEndpoint = "groups"
+	MaxLevel       = uint64(5)
+	MinLevel       = uint64(1)
+)
 
 type assignRequest struct {
 	Type    string   `json:"type,omitempty"`
 	Members []string `json:"members"`
 }
 
+type Member struct {
```

Comment:
I am not sure, but I think these kind of hig-level struct types are defined elsewhere. Probably in the `sdk.go`.